### ISSUE TITLE
nginx "OCSP stapling failed" errors

### ISSUE THREAD

**Issue by Southparkfan** [miraheze/puppet#10]
This is thousands (if not millions) of times in error.log:
```
2015/09/10 16:49:41 [error] 5352#0: OCSP_basic_verify() failed (SSL: error:27069065:OCSP routines:OCSP_basic_verify:certificate verify error:Verify error:unable to get issuer certificate) while requesting certificate status, responder: ocsp2.globalsign.com
```
Should be fixed.

**Comment by JohnFLewis** [comment]
Seems its failing to verify anuwiki.com and spiral.wiki against GlobalSign. Looking into it more now.

**Comment by JohnFLewis** [comment]
resolved with f3dad3a1e7f51a4c70f0fe13850a00de474c47c9
